### PR TITLE
Documentation revision #1010

### PR DESCRIPTION
**Pull request by oraNod**
#1010

### CODE REVIEW

File: documentation/asciidoc/topics/ref_logging.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+|===
+
+.Log levels
+[%header,cols=2*]
+|===
+|Log level
+|Description
+
+|`TRACE`
```
Comment (dmvolod) [inline documentation/asciidoc/topics/ref_logging.adoc:28]: Values for log level inside Infinispan CR should be lower case.

Comment (oraNod) [reply documentation/asciidoc/topics/ref_logging.adoc]: nice catch. thanks @dmvolod

File: documentation/asciidoc/topics/proc_allocating_storage.adoc [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+
+[IMPORTANT]
+====
+If available container storage is less than the amount of available memory, data loss can occur.
+====
+
+.Procedure
+
+. Allocate storage resources with the `spec.service.container.storage` field.
+. Configure the `ephemeralStorage` and `storageClassName` fields as required.
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_allocating_storage.adoc:16]: @oraNod probably I missed context, but both `ephemeralStorage` and `ephemeralStorage` are optional for `spec.service.container.storage` depends customer needed

File: documentation/asciidoc/stories/assembly_creating_clusters.adoc [hunk 3]
```diff
@@ -1,25 +1,15 @@
...
-ifdef::community[]
-* Have an `oc` or a `kubectl` client.
-endif::community[]
-//Downstream
-ifdef::downstream[]
-* Have an `oc` client.
-endif::downstream[]
+:context: infinispan-cr
+= Getting started with Infinispan CR
+`Infinispan` Custom Resource (CR) instructs {ispn_operator} to create and manage {brandname} clusters on {k8s}.
```
Comment (ryanemerson) [inline documentation/asciidoc/stories/assembly_creating_clusters.adoc:5]: I'm not sure "instructs" is the correct term here. It's more of a specification that is reconciled by the operator in order to create a cluster matching those requirements. Not sure how to word that though :smile:

File: documentation/asciidoc/stories/assembly_setting_up_xsite.adoc [hunk 4]
```diff
@@ -2,18 +2,44 @@ ifdef::context[:parent-context: {context}]
 [id='setting-up-xsite']
 :context: cross-site
 = Setting up cross-site replication
-Cross-site replication allows you to back up data from one {brandname} cluster to another.
+Ensure service availability, and guarantee business continuity, with {ispn_operator} by configuring cross-site replication to back up data between {brandname} clusters.
```
Comment (ryanemerson) [inline documentation/asciidoc/stories/assembly_setting_up_xsite.adoc:5]: "guarantee business continuity" I think we should remove this as it sounds like marketing, not technical docs.

File: documentation/asciidoc/stories/assembly_setting_up_xsite.adoc [hunk 5]
```diff
@@ -2,18 +2,44 @@ ifdef::context[:parent-context: {context}]
...
-You can use both automatic and manual connections for {brandname} clusters in the same `Infinispan` CR.
-However, you must ensure that {brandname} clusters establish connections in the same way at each site.
-====
+include::{topics}/proc_applying_cluster_roles_xsite.adoc[leveloffset=+3]
+include::{topics}/proc_creating_kubernetes_site_secrets.adoc[leveloffset=+3]
 
-include::assembly_automatically_connecting_clusters.adoc[leveloffset=+1]
-include::assembly_manually_connecting_clusters.adoc[leveloffset=+1]
+=== {openshiftshort} clusters
+Create and exchange service account tokens if you run {ispn_operator} on vanilla {k8s} or minikube.
```
Comment (ryanemerson) [inline documentation/asciidoc/stories/assembly_setting_up_xsite.adoc:20]: Don't you have to exchange tokens on openshift as well?

File: documentation/asciidoc/topics/con_xsite_replication.adoc [hunk 6]
```diff
@@ -1,5 +1,7 @@
 [id='cross-site-replication_{context}']
-= Cross-site replication with {ispn_operator}
+= Automatically forming cross-site views with {ispn_operator}
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/con_xsite_replication.adoc:2]: Is it really "Automatic"? You have to configure the URLS in the Infinispan CR and  the cache definitions as well as exchanging tokens.

Maybe "Managed" is a more appropriate term?

File: documentation/asciidoc/topics/ref_persistent_cache_store.adoc [hunk 7]
```diff
@@ -0,0 +1,14 @@
+[id='persistent-volume-claims_{context}']
+= Persistent volume claims
+{ispn_operator} creates a persistent volume claim (PVC) and mounts container storage at: +
+`{pv_path}`
+
+.Caches
+When you create caches, {brandname} permanently stores them so they are available after cluster restarts.
```
Comment (ryanemerson) [inline documentation/asciidoc/topics/ref_persistent_cache_store.adoc:7]: "permanently stores their configuration"? The current wording might be missinterpreted as the entire cache, including it's data, being permanently stored.

**Review by ryanemerson — CHANGES_REQUESTED** [review]

File: documentation/asciidoc/stories/assembly_setting_up_xsite.adoc (hunk 5 above)
Comment (oraNod) [reply documentation/asciidoc/stories/assembly_setting_up_xsite.adoc:20]: whoops. typo! thanks for catching this. it should be for OpenShift only.

**Review by rigazilla — APPROVED** [review]
I like the chapters reorg!
overall looks good to me

**Review by dmvolod — APPROVED** [review]

**Review by ryanemerson — APPROVED** [review]

**Comment by oraNod** [comment]
Squashing and merging after approvals. Thanks all!

**Pull request merged**